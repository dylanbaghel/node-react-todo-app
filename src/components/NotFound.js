import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => {
    return (
        <div className="container mx-auto">
            <div className="card card-body">
                <h2>404 | Not Found <Link to="/" >Go Home</Link></h2>
            </div>
        </div>
    );
};

export default NotFound;