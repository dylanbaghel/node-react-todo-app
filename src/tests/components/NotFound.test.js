import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import NotFound from './../../components/NotFound';

test('should render Header Component correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<NotFound />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});