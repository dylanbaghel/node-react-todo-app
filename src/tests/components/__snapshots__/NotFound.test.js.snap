// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header Component correctly 1`] = `
<div
  className="container mx-auto"
>
  <div
    className="card card-body"
  >
    <h2>
      404 | Not Found 
      <Link
        replace={false}
        to="/"
      >
        Go Home
      </Link>
    </h2>
  </div>
</div>
`;
